chainerx_cc/chainerx/routines/creation.h

```diff
@@ -120,4 +122,11 @@ Array Linspace(
         const nonstd::optional<Dtype>& dtype = nonstd::nullopt,
         Device& device = GetDefaultDevice());
 
+enum class MeshgridIndexingMode {
+    xy = 1,
+    ij,
```

Comment:
How about `MeshgridIndexingMode::{kCartesian,kMatrix}` ? A bit clearer I think. And prefixing with `k` is per convention.

Reply:
Done.